rolling-upgrade-clusters/README.adoc

```diff
@@ -3,25 +3,56 @@
 = Upgrading Kubernetes cluster on AWS
 
```

Comment:
It might be worth discussing different strategies for upgrading cluster. For example, creating a new cluster and switching the traffic to the new one and then terminate the existing one. And then one below is in action already.

Reply:
I like it. Just for clarity - currently i have in-place upgrade strategy and you are thinking of migration scenarios? what would be the name of it though? blue/green?

Reply:
yep, let's call this as in-place upgrade instead and create an issue for blue/green (if not already there)